.github/workflows/canary_ota.yml

```diff
@@ -17,8 +17,8 @@ on:
 env:
   ESP32_CANARY_ROBOT: ${{ secrets.ESP32_CANARY_ROBOT }}
   ESP32_CANARY_ROBOT_PART_ID: ${{ secrets.ESP32_CANARY_ROBOT_PART_ID }}
-  ESP32_CANARY_API_KEY_2: ${{ secrets.ESP32_CANARY_API_KEY_2 }}
-  ESP32_CANARY_API_KEY_ID_2: ${{ secrets.ESP32_CANARY_API_KEY_ID_2 }}
+  ESP32_CANARY_API_KEY: ${{ secrets.ESP32_CANARY_API_KEY }}
```

Comment:
If we no longer need `KEY_*_2`, please remember to remove them from the repository settings after this merges.